repl/scala-2.11/src/main/scala/org/apache/spark/repl/SparkIMain.scala

```diff
@@ -1129,7 +1129,7 @@ class SparkIMain(@BeanProperty val factory: ScriptEngineFactory, initialSettings
 
     def apply(line: String): Result = debugging(s"""parse("$line")""")  {
       var isIncomplete = false
-      currentRun.reporting.withIncompleteHandler((_, _) => isIncomplete = true) {
+      currentRun.parsing.withIncompleteHandler((_, _) => isIncomplete = true) {
```

Comment:
Just curious, was this a break across maintenance releases? or is this basically not a public Scala API anyway?

Are there any potential gotchas in updating? You know more than I would. I tend to favor updating as well to get fixes.